index.html

```diff
@@ -2,12 +2,56 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Hello Microverse!</title>
 </head>
 <body>
     <header>
-        <h1>Server is working!</h1>
+      <nav>
+        <div class="navbar">
+          <a class="logo" href="#">My Logo</a>
+          <a class="menu" href="#">
+            <div class="one"></div>
+            <div class="two"></div>
+            <div class="three"></div>
+          </a>
+       </div>
```

Comment:
- [ ] Kindly wrap your toolbar in a `<header>` tag instead of a `<div>`  Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.